packages/glimmer-test-helpers/lib/environment.ts

```diff
@@ -507,6 +519,10 @@ class EmberishCurlyComponentManager implements ComponentManager<EmberishCurlyCom
     }
   }
 
+  didRenderLayout(component: EmberishCurlyComponent, bounds: Bounds) {
+    component.bounds = bounds;
```

Comment:
The changes we are commenting on are in the `glimmer-test-helpers` package and are intended to roughly emulate what Ember's environment will do.

> Let's not add a conflict with user land props

@krisselden is basically saying that we should not stomp on properties that may exist already on the `component` passed in here.  We could use a private symbol, an underscored property, or even put information on a weakmap/meta object instead.

> Also shaping issues with this

Changing an objects shape (adding new methods or properties) after its constructor has completed (aka after `init`) makes the specific "class" have many possible shapes. Since the shape of the object is taken into consideration by V8 when optimizing this changing of shape can trigger deoptimizations when previously optimized code now sees a different shape.